packages/set/test/index.test.ts

```diff
@@ -100,33 +100,39 @@ describe("ReactiveSet", () => {
       const dispose = createRoot(dispose => {
         createEffect(() => {
           const run: number[] = [];
-          let i = 0;
           for (const key of fn(set)) {
             run.push(key);
-            if (i === 2) break; // don't iterate over all keys
-            i += 1;
           }
           captured.push(run);
```

Comment:
Can you make another test instead of modifying the previous one? One focusing just on what changed. This one is too big already.